packages/@atjson/editor/src/components/editable-link.ts

```diff
@@ -0,0 +1,62 @@
+import EditableComponent from '../mixins/editable-component';
+import LinkEditor from './link-editor';
+
+if (!window.customElements.get('link-editor')) {
+  window.customElements.define('link-editor', LinkEditor);
+}
+
+export default class EditableLink extends EditableComponent {
+
+  static template = `<div class="controls"><link-editor></link-editor></div><a class="text-link" target="_blank"><slot></slot></a>`;
+
+  static observedAttributes = ['url', 'nofollow'];
+
+  static style = EditableComponent.style + `
```

Comment:
I wanted something more elegant here, but was somewhat stymied by my TypeScript knowledge (calling super on class accessor methods wasn't working). I don't know glamourous, but I'm really happy to adopt anything else (but we should definitely support stylesheet inheritance to avoid boilerplate)